### DISCUSSION TITLE
vercel一键部署后是这样的，怎么解决啊

### DISCUSSION THREAD

**Discussion by syuchua** [ChatGPTNextWeb/NextChat#3170] [Q&A]
![Screenshot_2023-11-07-16-06-07-82_a252b927494330cdc2c8ba3b3f952e5e.jpg](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/131693964/c1d6bab8-0cb5-4469-9797-1f364be80080)

**Comment by H0llyW00dzZ** [comment]
https://platform.openai.com/account/billing/overview

**Comment by leeadee** [comment]
API问题？

**Reply by H0llyW00dzZ** [reply]
yes

**Reply by restmine** [reply]
> yes
It whether means  I must have enough dollars in my account to use the webapp
